components/PostCard.tsx

```diff
@@ -28,22 +58,36 @@ const toBase64 = (str: string) => Buffer.from(str).toString("base64");
 const PostCard = ({ post }: Props) => {
   const dispatch = useAppDispatch();
 
+  const classNameOfPost =
+    isPostADraft(post) === true || isPostInTheFuture(post) === false
+      ? "posts-list-block posts-list-block-draft-or-future"
+      : "posts-list-block";
+
   return (
     <li>
-      <div className="posts-list-block">
+      {isPostADraft(post) ? <DraftPostMark /> : null}
+      {isPostInTheFuture(post) === false ? <FuturePostMark /> : null}
+      <div className={classNameOfPost}>
         <div className="content">
           <Link href={`/post/${post.slug}`}>
             <a className="post-img">
-              <Image
-                src={post.featuredImage ?? "/post-images/placeholder.png"}
-                alt="blog post image"
-                layout="fill"
-                objectFit="cover"
-                placeholder="blur"
-                blurDataURL={`data:image/svg+xml;base64,${toBase64(
-                  shimmer(378, 378)
-                )}`}
-              />
+              {post.featuredImage ? (
+                <Image
+                  src={post.featuredImage}
+                  alt="blog post image"
+                  layout="fill"
+                  objectFit="cover"
+                  placeholder="blur"
+                  blurDataURL={`data:image/svg+xml;base64,${toBase64(
+                    shimmer(378, 378)
+                  )}`}
+                  style={{ filter: "grayscale(50%)" }}
```

Comment:
![image](https://user-images.githubusercontent.com/46556479/197393969-bfa78176-4f49-474f-9615-66254d076b03.png)
Why every picture is greyed out right now?

It should be only for upcoming posts